data/create-project-de.sql

```diff
@@ -330,7 +330,7 @@ Viele Grüße
 Das Team des ePartool'
     ),
     (
-        'notification_new_input_discussion_contrib_created',
+        'notification_new*input_discussion_contrib_created',
```

Comment:
This is an id of the email template and it must not be changed.

Reply:
Oops, that is a convincing comment ;-) Thanks for pointing it out. Corrections will follow.